docs/lib/storage/fragments/ios/getting-started.md

```diff
@@ -89,41 +88,44 @@ open <YOURAPP>.xcworkspace
 
 Initialize `AWSMobileClient`, `Amplify`, and `AWSS3StoragePlugin`.
 
-Add the following imports to the top of your `AppDelegate.swift` file
-    ```swift
-    import Amplify
-    import AWSMobileClient
-    import AmplifyPlugins
-    ```
-Add the following code to your AppDelegate's `application:didFinishLaunchingWithOptions` method
-
-    ```swift
-    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-        // Override point for customization after application launch.
-
-        AWSMobileClient.default().initialize { (userState, error) in
-            guard error == nil else {
-                print("Error initializing AWSMobileClient. Error: \(error!.localizedDescription)")
-                return
-            }
-            print("AWSMobileClient initialized, userstate: \(userState)")
-        }
+Add the following imports to the top of your `AppDelegate.swift` file:
+
+```swift
+import Amplify
+import AWSMobileClient
+import AmplifyPlugins
+```
+
+Add the following code to your AppDelegate's `application:didFinishLaunchingWithOptions` method:
 
-        let storagePlugin = AWSS3StoragePlugin()
-        do {
-            try Amplify.add(plugin: storagePlugin)
-            try Amplify.configure()
-            print("Amplify configured with storage plugin")
-        } catch {
-            print("Failed to initialize Amplify with \(error)")
+```swift
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+    // Override point for customization after application launch.
+
+    AWSMobileClient.default().initialize { (userState, error) in
+        guard error == nil else {
+            print("Error initializing AWSMobileClient. Error: \(error!.localizedDescription)")
+            return
         }
+        print("AWSMobileClient initialized, userstate: \(userState)")
+    }
 
-        return true
+    let storagePlugin = AWSS3StoragePlugin()
```

Comment:
can move this to `try Amplify.add(plugin: AWSS3StoragePlugin()`

Reply:
done!